openedx/adg/lms/webinars/models.py

```diff
@@ -28,10 +32,11 @@ class WebinarQuerySet(models.QuerySet):
 
     def delete(self):
         cancelled_upcoming_webinars = self.filter(
-            status=Webinar.UPCOMING).select_related('presenter').prefetch_related('co_hosts', 'panelists')
+            is_cancelled=False, start_time__gte=now()
```

Comment:
The filter `is_cancelled=False, start_time__gte=now()` is used 4 times in the code. Can we somehow come up with a manager that includes this logic and hence help us avoid code duplication. 

We should do the same with `is_cancelled=False, start_time__lt=now()`, although it is used only once, just a better practice.